src/olympia/files/tests/test_utils_.py

```diff
@@ -490,3 +490,13 @@ def test_bump_version_in_package_json(file_obj):
         utils.update_version_number(file_obj, '0.0.1.1-signed')
         parsed = utils.parse_xpi(file_obj.file_path)
         assert parsed['version'] == '0.0.1.1-signed'
+
+
+def test_bump_version_in_manifest_json(file_obj):
```

Comment:
Ah, this is related to https://github.com/mozilla/olympia/pull/1622/files#diff-910d9d00817510d6161315516feecfcbR748 ?